icons/package.svg

```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="16.5" y1="9.4" x2="7.5" y2="4.21" />
+  <line x1="16.5" y1="9.4" x2="7.9" y2="4.4" />
```

Comment:
Still not OK, I'm afraid.
![image](https://user-images.githubusercontent.com/17746067/228317597-b65c3a7f-d109-43e2-a51b-9e4ff35017da.png)

Please just believe me and apply the suggestion already. :D
```suggestion
  <line x1="16.5" y1="9.4" x2="7.55" y2="4.24" />
```